src/Exception/EntityNotFoundException.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Linio\Common\Exception;
+
+use Throwable;
+
+class EntityNotFoundException extends ClientException
+{
+    public function __construct(
+        string $token = ExceptionTokens::ENTITY_NOT_FOUND,
+        string $message = ExceptionTokens::ENTITY_NOT_FOUND,
```

Comment:
The `$token` is required only in the project's `DomainException`, right? For the subclass it's optional.
Maybe we could make the `$token` an internal exception attribute and the subclass only optionally receiving a `$message`.